defaults/main.yml

```diff
@@ -14,11 +14,19 @@ kafka_scala_version: "2.11"
 kafka_java_enabled: true
 # set false if the role should not handle the java dependency
 
-kafka_mirror: http://apache.mirrors.tds.net/kafka
-kafka_url: "{{ kafka_mirror }}/{{ kafka_version }}/kafka_{{ kafka_scala_version }}-{{ kafka_version }}.tgz"
+# Use this selector to use a mirror for a "current" version or an "archived" one
+kafka_mirror_selector: "current"
+kafka_mirror:
+  current: http://apache.mirrors.tds.net/kafka
```

Comment:
Seems like overkill for this: https://stackoverflow.com/questions/38592289/downloading-file-from-the-right-apache-mirror-with-wget

Just FYI.

Reply:
It seems that `https://archive.apache.org/dist/kafka/` has everything actually.

Reply:
```
kafka_mirror: https://www.apache.org/dyn/closer.cgi?action=download&filename=/kafka
```
Would work for the tar.gz, but unfortunately it doesn't have the signatures.